frontend/src/pages/LogsPage/LogsTable/LogsTable.tsx

```diff
@@ -218,101 +275,89 @@ const LogsTableInner = ({
 	}, [])
 
 	return (
-		<div style={{ height: `${totalSize}px`, position: 'relative' }}>
-			{paddingTop > 0 && <Box style={{ height: paddingTop }} />}
-
-			{virtualRows.map((virtualRow) => {
-				const row = rows[virtualRow.index]
-				const log = row.original.node
-				const matchedAttributes = findMatchingLogAttributes(
-					queryTerms,
-					{
-						...log.logAttributes,
-						level: log.level,
-						message: log.message,
-						secure_session_id: log.secureSessionID,
-						service_name: log.serviceName,
-						service_version: log.serviceVersion,
-						source: log.source,
-						span_id: log.spanID,
-						trace_id: log.traceID,
-					},
-				)
+		<Table height="full" noBorder>
+			<Table.Head>
+				<Table.Row gridColumns={GRID_COLUMNS}>
+					<Table.Header>Timestamp</Table.Header>
+					<Table.Header>Level</Table.Header>
+					<Table.Header>Body</Table.Header>
+				</Table.Row>
+				{enableFetchMoreLogs && (
+					<Table.Row>
+						<Box width="full">
+							<AdditionalFeedResults
+								more={moreLogs}
+								type="logs"
+								onClick={() => {
+									clearMoreLogs()
+									handleAdditionalLogsDateChange()
+								}}
+							/>
+						</Box>
+					</Table.Row>
+				)}
+			</Table.Head>
+			<Table.Body
+				forwardRef={bodyRef}
```

Comment:
Nit, but what do you think about using `forwardRef` inside `Table.Body` to pass the ref rather than a custom prop?